alot/commands/globals.py

```diff
@@ -78,8 +77,7 @@ def apply(self, ui):
 
         if ui.db_was_locked:
             msg = 'Database locked. Exit without saving?'
-            if (yield ui.choice(msg, select='yes', cancel='no',
-                                msg_position='left')) == 'no':
+            if (yield ui.choice(msg, select='yes', cancel='no')) == 'no':
```

Comment:
Why did you change the `msg_position`? Is it related to the select stuff?

And why do you still `select='yes'` here?

What do I need to do in order to get to this prompt (to test it)?